libvirt/tests/src/virsh_cmd/domain/virsh_destroy.py

```diff
@@ -74,6 +100,49 @@ def run(test, params, env):
                                unprivileged_user=unprivileged_user,
                                uri=uri, debug=True).exit_status
         output = ""
+
+        if start_destroy_times:
+            status = 0
+            try:
+                # RHEL9 will config virtqemud.service
+                modify_service = "virtqemud.service"
+                if distro.detect().name == 'rhel' and \
+                        int(distro.detect().version) < 9:
+                    modify_service = "libvirtd.service"
+                service_file = "/usr/lib/systemd/system/{}"\
+                               .format(modify_service)
+                LOG.debug("Modify service: {}".format(service_file))
+
+                # Backup original libvirtd.service
+                backup_file = os.path.join(data_dir.get_tmp_dir(),
+                                           "{}-bak".format(modify_service))
+                shutil.copy(service_file, backup_file)
+
+                # Decrease domain number to speed up test
+                modify_virtual_daemon(service_file, limit_nofile)
+                process.run("systemctl daemon-reload")
+                if "virtqemud.service" in modify_service:
+                    utils_split_daemons.virtqemud_restart()
+                else:
+                    utils_libvirtd.libvirtd_restart()
```

Comment:
I would recommend using utils_libvirtd.Libvirtd() directly.
```suggestion
              utils_libvirtd.Libvirtd('virtqemud').restart()  
```